src/features/home/Home.jsx

```diff
@@ -1,3 +1,17 @@
+import { NavLink } from "react-router-dom";
+
 export default function Home() {
-    return (<></>)
-}
\ No newline at end of file
+  return (
+    <>
+      <h2>Weclome to Fullstack University!</h2>
+      <img
+        src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/e8/Univerza_v_Ljubljani_01.jpg/640px-Univerza_v_Ljubljani_01.jpg"
+        alt="Fullstack University"
+      />
+      <p>Please use the below links to navigate:</p>
+      <NavLink to="/departments">Find a Department</NavLink>
+      <NavLink to="/professors">Find a Professor</NavLink>
+      <NavLink to="/auth">Login or Register for an Account</NavLink>
```

Comment:
Can we do "/login" instead of "/auth"? I think that would be more familiar for users to recognize in a URL (also is currently the route that's linked to our` <Auth>` component